[{"ContainerToolsException.Message":"golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:54 +0x92","ContainerToolsException.ErrorCode":"DT1001","ContainerToolsException.StackTrace":"   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Docker.DockerComposeClient.<ExecuteAsync>d__13.MoveNext() in D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerComposeClient.cs:line 253\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Docker.DockerComposeClient.<DownAsync>d__8.MoveNext() in D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerComposeClient.cs:line 115\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Docker.DockerWorkspace.<ComposeDownContainersAsync>d__31.MoveNext() in D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerWorkspace.cs:line 418\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Docker.BuildTasks.DockerBuildTask.<ExecuteAsync>d__38.MoveNext() in D:\\a\\_work\\1\\s\\src\\Microsoft.Docker.BuildTasks\\DockerBuildTask.cs:line 87","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.CommandLine.Program":"docker-compose","ContainerToolsException.CommandLine.TelemetrySafeArguments":"","ContainerToolsException.CommandLine.ExitCode":"2","ContainerToolsException.CommandLine.ErrorOutput":"panic: runtime error: invalid memory address or nil pointer dereference\r\n[signal 0xc0000005 code=0x0 addr=0x28 pc=0x1b805b5]\r\ngoroutine 41 [running]:\r\ngithub.com/docker/compose/v2/pkg/compose.(*composeService).getServiceImages(0x0, {0x1, 0x0, 0x0, {0xc00003c1b0, 0x5}, 0x0}, {0xc0000363c0, 0x21})\r\n\tgithub.com/docker/compose/v2/pkg/compose/down.go:149 +0x75\r\ngithub.com/docker/compose/v2/pkg/compose.(*composeService).ensureImagesDown(0xc0003ca080, {0x206b2c8, 0xc0003c8570}, {0xc0000363c0, 0x21}, {0x1, 0x0, 0x0, {0xc00003c1b0, 0x5}, ...}, ...)\r\n\tgithub.com/docker/compose/v2/pkg/compose/down.go:119 +0xc8\r\ngithub.com/docker/compose/v2/pkg/compose.(*composeService).down(0xc0003ca080, {0x206b2c8, 0xc0003c8570}, {0xc0000363c0, 0x21}, {0x1, 0x0, 0x0, {0xc00003c1b0, 0x5}, ...})\r\n\tgithub.com/docker/compose/v2/pkg/compose/down.go:85 +0x4b4\r\ngithub.com/docker/compose/v2/pkg/compose.(*composeService).Down.func1({0x206b2c8, 0xc0003c8570})\r\n\tgithub.com/docker/compose/v2/pkg/compose/down.go:39 +0x85\r\ngithub.com/docker/compose/v2/pkg/progress.Run.func1({0x206b2c8, 0xc0003c8570})\r\n\tgithub.com/docker/compose/v2/pkg/progress/writer.go:61 +0x27\r\ngithub.com/docker/compose/v2/pkg/progress.RunWithStatus.func2()\r\n\tgithub.com/docker/compose/v2/pkg/progress/writer.go:82 +0x87\r\ngolang.org/x/sync/errgroup.(*Group).Go.func1()\r\n\tgolang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57 +0x67\r\ncreated by golang.org/x/sync/errgroup.(*Group).Go\r\n\tgolang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:54 +0x92"}]